.github/workflows/abi.yaml

```diff
@@ -77,6 +77,7 @@ jobs:
             pg: 16
             builder: ${{ fromJson(needs.config.outputs.pg16_latest) }}-alpine3.18
             tester: ${{ fromJson(needs.config.outputs.pg16_abi_min) }}-alpine
+            ignores: pg_dump_unprivileged
```

Comment:
Could you add a comment to this line on why this is ignored (i.e., something similar to the commit message would be great)?